proto/gateway/service.proto

```diff
@@ -46,6 +47,20 @@ message GetFeatureFlagsResponse {
   bool force_update = 5;
 }
 
+message GetSegmentUsersRequest {
+  repeated string segment_ids = 1;
```

Comment:
This list contains the IDs of the segment cache in the SDK. It will be used to compare and check if one or more segments have been deleted on the server.